### PR TITLE
Añado la historia del vómito

### PR DESCRIPTION
**Pull request by franciscodelicado**
Review this contribution  @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2451)

### CODE REVIEW

File: espanol/comida/comida.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Aguantose las náuseas, y tras enjugarse el sudor, pensó que nuevas le traería el capitán de las tropas:
+
+[El ejercito había vencido al enemigo](victoria/victoria.md)
+[Una orda de ogros había atravesado la frontera norte](ogros/ogros.md)
```
Comment (casey-collab) [inline espanol/comida/comida.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/comida/comida.md (hunk 1 above)
Comment (franciscodelicado) [inline espanol/comida/comida.md]: Review again the changes @casey-collab

**Comment by franciscodelicado** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**